README.md

```diff
@@ -18,21 +18,20 @@ yarn add hue-map
 ## Usage
 
 ```js
-import createPalette from 'hue-map'
+import { createPalette } from 'hue-map'
 
 const myPalette = createPalette({
   map: 'viridis',
   steps: 3,
-  format: 'cssHex',
 })
 
-console.log(myPalette)
-// ['#440154FF', '#21908DFF', '#FDE725FF']
+console.log(myPalette.format('cssHex'))
```

Comment:
This needs to be documented properly in the table below, i.e. format should no longer be in the options table